bedrock/firefox/redirects.py

```diff
@@ -43,7 +43,8 @@ def firefox_mobile_faq(request, *args, **kwargs):
     redirect(r'^firefox/accountmanager/?$', 'https://developer.mozilla.org/Persona'),
 
     # bug 841846
```

Comment:
Nit: please append the bug number along with the existing one here so we have some paper trail :)